.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -6,6 +6,7 @@ on:
       - "apps/codeforafrica/**"
       - "Dockerfile.codeforafrica"
       - ".github/workflows/codeforafrica-deploy-review.yml"
+    types: [opened, reopened, edited, closed, synchronize]
```

Comment:
For the slow ones such as myself, what does this add? Or put another way, what are the default `types`?

Reply:
@kilemensi From the [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened. To trigger workflows by different activity types, use the types keyword.

Reply:
Nice... can we add this as a comment above the types line? i.e. We need `closed` `edited` in addition to the default events; `opened`, `synchronize`, ad `reopened`.

What does edited do?

Reply:
Good catch @kilemensi. Edited runs when a pull request's description or labels are edited. I've removed it.